Library/Formula/brotli.rb

```diff
@@ -0,0 +1,27 @@
+class Brotli < Formula
+  desc "Generic-purpose lossless compression algorithm by Google."
+  homepage "https://github.com/google/brotli"
+  url "https://github.com/google/brotli/archive/v0.2.0.tar.gz"
+  sha256 "634d1089ee21b35e0ec5066cb5e44dd097e04e679e1e8c50bffa2b0dc77c2c29"
+
+  head "https://github.com/google/brotli"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+  conflicts_with "bro", :because => "Both install a `bro` binary"
```

Comment:
It’d be awesome if you could add a similar line in the `bro` formula (in a separate commit), e.g.:
```rb
conflicts_with "brotli", :because => "Both install a `bro` binary"
```